### PR TITLE
Tidying up History page part 1

### PR DESCRIPTION
**Pull request by nhyty**
See #65 and #474. Hopefully i have added the images right, @wjdp you just need to prettify.

### CODE REVIEW

File: history.md [hunk 1]
```diff
@@ -27,22 +27,20 @@ Nottingham's Everyman Club was formed in 1958, to encourage students and local p
...
-
-In 2012 the committee voted to alter the name of the theatre from "The New Theatre" to "The Nottingham New Theatre".
+![Foyer Extension](http://photos.newtheatre.org.uk/2011-12/Matthew-Bannister-Visit-2012/i-VgSsh2w/0/M/428161_10150669394401460_1300692780_n-M.jpg)
 
 
-## New Theatre Alumni Events
-
-16th November 2013 - IMPACT Fundraising Event held at the New Theatre to launch and promote the sponsor-a-seat campaign. About 50 alumni were in attendance. They were offered a tour around the newly refurbished building, and given a short 20 minute excerpt from this week's show!
+Until recently, the New Theatre was contained within the Archaeology and Classics building. The summer of 2012 saw an extensive redevelopment of the building housing the New Theatre. The Archaeology and Classics building was demolished from the site, leaving the New Theatre as its own freestanding building. Parts of the old building were retained, and repurposed as new rehearsal rooms, a studio space, storage rooms and a new constume cupboard. The foyer was also significantly extended.
```
Comment (wjdp) [inline history.md]: Not really sure why this was moved here?

**Comment by wjdp** [comment]
Yep yep, doing now

**Comment by nhyty** [comment]
I didn't fail!

**Comment by wjdp** [comment]
One of the image URLs was wrong :stuck_out_tongue:

**Comment by nhyty** [comment]
Damn it!

**Pull request merged**